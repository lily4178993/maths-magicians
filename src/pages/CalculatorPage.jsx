import React from 'react';
import { Calculator } from '../components';

const CalculatorPage = () => (
  <section className="section-calculator">
    <h2 className="section-title fade-in-left">Let&apos;s do some maths!</h2>
    <Calculator />
  </section>
);

export default CalculatorPage;
