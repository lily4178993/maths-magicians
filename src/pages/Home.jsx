import React from 'react';

const Home = () => (
  <section className="section-home fade-in-top">
    <h2 className="section-title fade-in-left">Welcome to our page!</h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, voluptatum
      eum! Tempora itaque saepe iure sed aut magni minima soluta laborum,
      tenetur voluptas eos, dignissimos, eligendi doloremque quo perspiciatis
      natus!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, voluptatum
      eum! Tempora itaque saepe iure sed aut magni minima soluta laborum,
      tenetur voluptas eos, dignissimos, eligendi doloremque quo perspiciatis
      natus!
    </p>
  </section>
);

export default Home;
