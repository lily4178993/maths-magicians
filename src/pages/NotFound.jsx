import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <section className="noFound fade-in-top">
    Page not found. Go to
    <Link to="/">Home</Link>
    page
  </section>
);

export default NotFound;
