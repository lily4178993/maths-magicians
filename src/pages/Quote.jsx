import React from 'react';
import { Quotes } from '../components';

const Quote = () => (
  <section className="section-quotes fade-in-top"><Quotes /></section>
);

export default Quote;
