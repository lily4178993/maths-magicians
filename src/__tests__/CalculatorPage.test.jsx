import React from 'react';
import { render } from '@testing-library/react';
import { CalculatorPage } from '../pages';

test('CalculatorPage component snapshot', () => {
  const { asFragment } = render(<CalculatorPage />);
  expect(asFragment()).toMatchSnapshot();
});
