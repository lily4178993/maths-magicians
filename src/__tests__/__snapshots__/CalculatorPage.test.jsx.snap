// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorPage component snapshot 1`] = `
<DocumentFragment>
  <section
    class="section-calculator"
  >
    <h2
      class="section-title fade-in-left"
    >
      Let's do some maths!
    </h2>
    <div
      class="calculator fade-in-top"
    >
      <div
        class="output"
      >
        <div
          class="output2"
        >
          0
        </div>
      </div>
      <div
        class="keyboard"
      >
        <button
          title="AC"
          type="button"
        >
          AC
        </button>
        <button
          title="+/-"
          type="button"
        >
          +/-
        </button>
        <button
          title="%"
          type="button"
        >
          %
        </button>
        <button
          class="orange"
          title="รท"
          type="button"
        >
          รท
        </button>
        <button
          title="7"
          type="button"
        >
          7
        </button>
        <button
          title="8"
          type="button"
        >
          8
        </button>
        <button
          title="9"
          type="button"
        >
          9
        </button>
        <button
          class="orange"
          title="x"
          type="button"
        >
          x
        </button>
        <button
          title="4"
          type="button"
        >
          4
        </button>
        <button
          title="5"
          type="button"
        >
          5
        </button>
        <button
          title="6"
          type="button"
        >
          6
        </button>
        <button
          class="orange"
          title="-"
          type="button"
        >
          -
        </button>
        <button
          title="1"
          type="button"
        >
          1
        </button>
        <button
          title="2"
          type="button"
        >
          2
        </button>
        <button
          title="3"
          type="button"
        >
          3
        </button>
        <button
          class="orange"
          title="+"
          type="button"
        >
          +
        </button>
        <button
          title="0"
          type="button"
        >
          0
        </button>
        <button
          title="."
          type="button"
        >
          .
        </button>
        <button
          class="orange"
          title="="
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
